crates/holochain_state/src/query.rs

```diff
@@ -308,16 +308,21 @@ impl<'stmt> Store for Txn<'stmt, '_> {
         }
     }
 
+    // Returns only actions that are referenced by at least one integrated dht op.
```

Comment:
I don't follow this logic. When an action is referenced by multiple ops, the validity of each op is independent. This means that it can depend which op gets checked first whether a dependent op can pass validation?